src/RoomsSDKAdapter.js

```diff
@@ -36,6 +39,10 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
 
     this.getRoomObservables = {};
     this.listenerCount = 0;
+
+    this.limit = 50;
```

Comment:
I would name this a little more explicitly. Not sure "limit" makes sense all on its own
```suggestion
    this.activityChunkSize = 50;
```

Reply:
Technically its the `limit` of items returned from the request. If we use `activityChunkSize` then it assumes we fetch `500` and chunk `50` Items in sets when invoked, I thought about this as it could save network requests but again it was not specified in the requirements. This is where developers have opinions on how things are named and fact its private so its kind of irrelevant.

Reply:
I disagree with your statement:

> This is where developers have opinions on how things are named and fact its private so its kind of irrelevant.

Your team's opinion is not irrelevant and it's a very arrogant attitude.
You're not writing code for yourself, you're writing code for all the people that will read it in the future, naming is very important.

For this case, the context of the the rooms adapter, what is limit? is this for rooms? activities? dates? You definitely need something more descriptive